packages/frontend/amp/components/KeyEvents.tsx

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import { css } from 'emotion';
-import { headline, textSans } from '@guardian/pasteup/typography';
-import { palette } from '@guardian/src-foundations';
+import { palette, headline, textSans } from '@guardian/src-foundations';
 // import { pillarPalette } from '@frontend/lib/pillars';
-import DownArrow from '@guardian/pasteup/icons/down-arrow.svg';
+import DownArrow from '@frontend/static/icons/down-arrow.svg';
```

Comment:
I think some of these SVGs are in the [`src-svgs` package](https://github.com/guardian/source-components/tree/master/packages/svgs) if that's helpful?

https://www.figma.com/file/Ai7AELHC6KCz38qKZkvuHo/Icons?node-id=55%3A2